cms/djangoapps/contentstore/tests/test_import.py

```diff
@@ -40,13 +39,14 @@ def setUp(self):
         self.client = Client()
         self.client.login(username=self.user.username, password=password)
 
-    def load_test_import_course(self, target_course_id=None, create_new_course_if_not_present=False):
+    def load_test_import_course(self, target_course_id=None, create_course_if_not_present=True, module_store=None):
```

Comment:
With old mongo, import_from_xml did not fail with create_course_if_not_present=False, even if the target course did not exist. It does with split.